zwave_js_server/model/value.py

```diff
@@ -87,6 +86,10 @@ def unit(self) -> Optional[str]:
         return self.data.get("unit")
 
     @property
+    def states(self) -> Optional[dict]:
+        """Return (optional) states."""
+        return self.data.get("states")
```

Comment:
We need to add this item to the TypedDict.

Reply:
will you do so or should I ?

Reply:
I did it now.